tests/integration/tasks/sync.ts

```diff
@@ -40,6 +40,19 @@ describe('exh tasks sync', () => {
     expect(logSpy).toHaveBeenCalledWith(chalk.green('Successfully created task', functionMock.functionConfig.name));
   });
 
+  it('Creates a Function with a default priority', async () => {
+    functionMock = functionRepositoryMock();
+    const taskConfigPath = await tempDirectoryManager.createTempJsonFile({ ...functionMock.functionConfig, defaultPriority: 12 });
+    await tempDirectoryManager.createTempJsFile('index', functionCode);
+
+    const logSpy = jest.spyOn(global.console, 'log');
+
+    await handler({ sdk: null, path: taskConfigPath });
+    expect(functionMock.findSpy).toHaveBeenCalledTimes(1);
+    expect(functionMock.createSpy).toHaveBeenCalledTimes(1);
+    expect(logSpy).toHaveBeenCalledWith(chalk.green('Successfully created task', functionMock.functionConfig.name));
```

Comment:
I might have missed something, but can we also check if the `defaultPriority` is actually forwarded to the task service by the CLI?
Something like the test below has, with `expect(functionMock.createSpy).toHaveBeenCalledWith(null, expect.objectContaining({ executionOptions: { defaultPriority: 12 } });`